easy_note/README.md

```diff
@@ -32,17 +32,30 @@ A simple note service built with `Kitex` and `Hertz` which is divided into three
 
 ### Catalog Introduce
 
-| catalog     | introduce               |
-|-------------|-------------------------|
-| handler     | HTTP handler            |
-| service     | business logic          |
-| rpc         | RPC call logic          |
-| dal         | DB operation            |
-| pack        | data pack               |
-| pkg/mw      | RPC middleware          |
-| pkg/consts  | constants               |
-| pkg/errno   | customized error number |
-| pkg/configs | SQL and Tracing configs |
+| catalog       | introduce               |
+|---------------|-------------------------|
+| hertz_handler | HTTP handler            |
+| service       | business logic          |
+| rpc           | RPC call logic          |
+| dal           | DB operation            |
+| pack          | data pack               |
+| pkg/mw        | RPC middleware          |
+| pkg/consts    | constants               |
+| pkg/errno     | customized error number |
+| pkg/configs   | SQL and Tracing configs |
+
+## Code Generate
```

Comment:
Is it better to use code generation and catalog introduction ?

Reply:
done